src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
"If you interpret it literally, it's actually wrong because it is not an
IOU issued by the destination"

Does 'counterparty' hold up as well as Dave is saying issuer does?

I have no strong preference on this subject. Needs further thought.

Maybe the only real advantage to omitting the counterparty would be less
clutter? I guess you might be able to argue it's more accessible and self
contained semantics.

An amount with counterparty B could mean different things when its used as
a source or destination amount where source account is B.

An amount with no counterparty? It's just an amount of some currency.

Hrrmmmm.